### DISCUSSION TITLE
Is there any temporal scale limit for these methods?

### DISCUSSION THREAD

**Discussion by muhshif** [btschwertfeger/python-cmethods#24] [Q&A]
In your opinion, can we apply these methods for sub-seasonal time scale (i.e., upto 1 month temporal scale) analysis of particular meteorological variables (eg. of temperature, rainfall, relative humidity, etc.)?

**Comment by btschwertfeger** [comment]
Adjustment using scaling-based methods such as LS, VS, and DM is feasible in any case. By default, these methods use the "time.month" parameter for grouping, which is based on long-term monthly averages. However, when dealing with a dataset containing only one month, the "group" parameter can be set to "None" to adjust the entire dataset based on the overall mean instead of the month-dependent mean.

To achieve satisfactory results, it is important that the control period data ("obs" and "simh") cover a period comparable to the scenario period. In other words, if the control period and scenario period are too far apart, bias correction results may be unreliable. For example, adjusting a single January from a scenario period requires taking the average of every single day of January over ten years or decades of control period data. In your case, it is crucial to use data spanning several years or decades to represent a single month accurately.

To disable the auto-grouping in scaling-based methods disable it like so:
```python
import xarray as xr
from cmethods import CMethods as cm

# Note: The data sets must contain the dimension "time" for the respective variable.
obsh = xr.open_dataset("path/to/reference_data-control_period.nc")
simh = xr.open_dataset("path/to/modeled_data-control_period.nc")
simp = xr.open_dataset("path/to/the_dataset_to_adjust-scenario_period.nc")
variable = "tas" # temperatures

ls_adjusted = cm.linear_scaling(
     obs=obs[variable], 
     simh=simh[variable], 
     simp=simp[variable], 
     kind="+", 
    group=None # this is important to disable the grouping
)
```

When the grouping is disabled, you can apply this method on custom time scales.

I hope this explanation clarifies any confusion.
